README.md

```diff
@@ -45,3 +45,20 @@ may well be used to fire requests to other services.
         * Only GET is supported at the moment not sure anything else is needed here
 
 
+#Docker
```

Comment:
Missing space between the hash and the "Docker" makes it not look like a title:

![image](https://user-images.githubusercontent.com/976274/44140232-61bd5b70-a071-11e8-8e94-b8bb885f12c0.png)

Reply:
I did this and also made it a sub-heading.